File: src/main/webapp/Front/src/app/model/activity.model.ts
export class ActivityModel {

  public title: string
  public message: string
  public module: string  
  public idUser: number
  public fullNameUser: string
  public createdAt: Date
  public dateString: string

  constructor(
    title?: string,
    message?: string,
    module?: string,
    idUser?: number,
    fullNameUser?: string,
    createdAt?: Date,
    dateString?: string) {
      this.title = title
      this.message = message
      this.module = module
      this.idUser = idUser
      this.fullNameUser = fullNameUser
      this.createdAt = createdAt
      this.dateString = dateString
  }
  
}

File: src/main/java/com/ear/core/model/ActivityModel.java
package com.ear.core.model;

import java.util.Date;

public class ActivityModel {

	private String title;
	private String message;
	private Integer idUser;
	private String fullNameUser;
	private String module;	
	private Date createdAt;

	public String getTitle() {
		return title;
	}

	public void setTitle(String title) {
		this.title = title;
	}
	
	public String getMessage() {
		return message;
	}

	public void setMessage(String message) {
		this.message = message;
	}

	public Integer getIdUser() {
		return idUser;
	}

	public void setIdUser(Integer idUser) {
		this.idUser = idUser;
	}

	public String getFullNameUser() {
		return fullNameUser;
	}

	public void setFullNameUser(String fullNameUser) {
		this.fullNameUser = fullNameUser;
	}		

	public String getModule() {
		return module;
	}

	public void setModule(String module) {
		this.module = module;
	}

	public Date getCreatedAt() {
		return createdAt;
	}

	public void setCreatedAt(Date createdAt) {
		this.createdAt = createdAt;
	}
	
}
